frontend/app/src/components/ui/link.tsx

```diff
@@ -5,6 +5,7 @@ type LinkProps = {
   to: string;
   children?: any;
   target?: string;
+  rel?: string;
   className?: string;
 };
```

Comment:
maybe we could complety replace that by LinkProps provided by "react-router-dom"